packages/core/src/components/datetime/test/error/datetime.e2e.ts

```diff
@@ -0,0 +1,76 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/datetime/test/error/index.html';
+
+test.describe('tds-datetime-error', () => {
+  test('renders disabled datetime component correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('verifies label, helper text, size, and clock icon for time component', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Check for the label text
+    const label = page.locator('.tds-datetime-label');
+    const helperText = page.locator('.tds-datetime-helper .tds-helper');
+    const clockIcon = page.locator('tds-icon[name="clock"]');
+    const dateTime = page.locator('tds-datetime');
+
+    await expect(label).toBeVisible();
+    await expect(label).toHaveText('Label text');
+
+    await expect(helperText).toBeVisible();
+    await expect(helperText).toHaveText('Helper text');
+
+    await expect(dateTime).toHaveAttribute('size', 'md');
+    await expect(clockIcon).toBeVisible();
+  });
+
+  test('helper text in error state has is red and has an error icon', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Check helper text color for specific shade of red
+    await expect(page.locator('.tds-datetime-helper .tds-helper')).toHaveCSS(
+      'color',
+      'rgb(255, 35, 64)',
+    );
+
+    const errorIcon = page.locator('tds-icon[name="error"]');
+    await expect(errorIcon).toBeVisible();
+  });
+
+  test('Clock icon focuses the time input', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    await page.click('input[type="time"]');
+
+    // Check if the time input is focused after clicking the icon
+    // This assumes the time input has a specific ID or class you can target
+    const isTimeInputFocused = await page.evaluate(() => document.activeElement.type === 'time');
```

Comment:
```suggestion
     const isTimeInputFocused = await page.evaluate(
      () => (document.activeElement as HTMLInputElement).type === 'time',
    );
```
Improvment for IDE and type issue.